src/Microsoft.AspNet.Routing/RouteValueDictionary.cs

```diff
@@ -31,23 +31,24 @@ public RouteValueDictionary()
         }
 
         /// <summary>
-        /// Creates a <see cref="RouteValueDictionary"/> initialized with the provided input value.
+        /// Creates a <see cref="RouteValueDictionary"/> initialized with the specified <paramref name="values"/>.
         /// </summary>
-        /// <param name="obj">An object to initialize the dictionary. The value can be of type
+        /// <param name="values">An object to initialize the dictionary. The value can be of type
         /// <see cref="IDictionary{TKey, TValue}"/> or <see cref="IReadOnlyDictionary{TKey, TValue}"/> or
         /// any other object.
```

Comment:
still unsure what the second sentence adds.  my preference is to remove but I'd like to hear @Eilon's thoughts.